api/jobs/gears.py

```diff
@@ -119,12 +132,17 @@ def insert_gear(doc):
     if doc.get("invocation-schema"):
         del(doc["invocation-schema"])
 
+    now = datetime.datetime.utcnow()
+
+    doc['created']  = now
+    doc['modified'] = now
+
     config.db.gears.insert(doc)
```

Comment:
Agreed that it doesn't fix our problem - it's exactly as bad as `b.OID` everywhere. I would argue that it adds less work to #654 though.